app/webhook.go

```diff
@@ -96,7 +97,11 @@ func (a *App) TriggerWebhook(c *request.Context, payload *model.OutgoingWebhookP
 	var body io.Reader
 	var contentType string
 	if hook.ContentType == "application/json" {
-		body = strings.NewReader(payload.ToJSON())
+		js, jsonErr := json.Marshal(payload)
+		if jsonErr != nil {
+			mlog.Warn("Failed to encode to JSON", mlog.Err(jsonErr))
+		}
+		body = strings.NewReader(string(js))
```

Comment:
Why `strings.Reader` over `bytes.Reader`?

Reply:
It was the quickest change that left the semantic unchanged. Had hit a couple of places where using bytes wasn't working as expected. This one seems safe though so I can update.